x-pack/plugins/search_playground/kibana.jsonc

```diff
@@ -0,0 +1,21 @@
+{
+  "type": "plugin",
+  "id": "@kbn/search_playground",
+  "owner": "@elastic/enterprise-search-frontend",
+  "plugin": {
+    "id": "searchPlayground",
+    "server": true,
+    "browser": true,
+    "configPath": [
+      "xpack",
+      "search_playground"
+    ],
+    "requiredPlugins": [
+      "navigation",
+      "security"
```

Comment:
What are we using `security` for?

Reply:
Using this one for getting userProfiles
`services.security?.userProfiles.getCurrent({ dataPath: 'avatar' }).then(setCurrentUserProfile);`